i18n.md

```diff
@@ -0,0 +1,230 @@
+
+## Environment variables
+
+You will need some environment variables, that you can set in the `sites/kit.svelte.dev/.env` file if you need them locally, or in your deployment environment if you need to [deploy](#deploying).
+
+### External links
+
+The docs contains references to external sites, such as the [SvelteKit site](https://learn.svelte.dev), the [tutorial site](https://learn.svelte.dev), or the [Github organization](https://github.com/sveltejs/svelte).
+
+In order to be able to easily customize these links, **we added environment variables** representing these links:
+- `PUBLIC_SVELTE_SITE_URL`: URL of the Svelte site
+- `PUBLIC_KIT_SITE_URL`: URL of the SvelteKit site
+- `PUBLIC_LEARN_SITE_URL`: URL of the tutorial site
+- `PUBLIC_GITHUB_ORG`: name of the Github organization/account that hosts this project
+
+At first, the `PUBLIC_SVELTE_SITE_URL` and `PUBLIC_LEARN_SITE_URL` variables might be just references to the original sites, and not provide any added value. But once you complete the translations of Kit and tutorial docs sites, you might need to use these variables to reference the URLs of your translated sites.
+
+The `PUBLIC_GITHUB_ORG` variable is mainly used to provide a link to your repo when inviting people to "Edit this page".
+
+We might want to use these variables in the following files:
+- `sites/kit.svelte.dev/src/routes/+error.svelte`
+- `sites/kit.svelte.dev/src/routes/+layout.svelte`
+- `sites/kit.svelte.dev/src/routes/+page.svelte`
+- `sites/kit.svelte.dev/src/routes/docs/[slug]/+page.svelte`
+- `sites/kit.svelte.dev/src/routes/home/Intro.svelte`
+
+> Be careful, this list might not be exhaustive.
+
+### Markdown processing
+
+As we have seen in the previous section, the docs contains references to external sites. You might want to also reference these links in the `.md` files, but this will require to **replace the environment variables at build time**.
+
+This can done in the `sites/kit.svelte.dev/src/lib/server/docs/index.js` file, like so:
+
+```js
+// ...
+const content = page.content
+	.replace(/PUBLIC_SVELTE_SITE_URL/g, PUBLIC_SVELTE_SITE_URL)
+	.replace(/PUBLIC_KIT_SITE_URL/g, PUBLIC_KIT_SITE_URL)
+	.replace(/PUBLIC_LEARN_SITE_URL/g, PUBLIC_LEARN_SITE_URL)
+	.replace(/PUBLIC_GITHUB_ORG/g, PUBLIC_GITHUB_ORG);
+// ...
+```
+
+## Translating the docs
+
+The core of the work to accomplish is in the `documentation/docs` folder.
+
+**We translated as much as possible in this folder**, even some strings in the code snippets so that even code examples are better understood.
+
+Do not forget to translate the `meta.json` files, which provide metadata like section titles.
+
+We did not feel necessary to translate the folder names, since it would 'only' affect URLs. This might have an bad influence on SEO. However, **the subsections titles in the `.md` files that you will eventually translate will break links** throughout your site. You will need to fix those before deployment.
```

Comment:
```suggestion
We did not feel necessary to translate the folder names, since it would 'only' affect URLs. This might have a bad influence on SEO. However, **the subsections titles in the `.md` files that you will eventually translate will break links** throughout your site. You will need to fix those before deployment.
```